src/store/reducers/storage.js

```diff
@@ -22,14 +22,15 @@ export const StorageTypes = {
 };
 
 const FETCH_STORAGE = createRequestActionTypes('storage', 'FETCH_STORAGE');
+// works exactly the same as FETCH_STORAGE, but does not trigger loading state
+const BG_FETCH_STORAGE = createRequestActionTypes('storage', 'BG_FETCH_STORAGE');
 const SET_INITIAL = 'storage/SET_INITIAL';
 const SET_FILTER = 'storage/SET_FILTER';
 const SET_VISIBLE_GROUPS = 'storage/SET_VISIBLE_GROUPS';
 const SET_STORAGE_TYPE = 'storage/SET_STORAGE_TYPE';
 
 const initialState = {
     loading: true,
-    wasLoaded: false,
```

Comment:
I'm not sure it's good idea because in all other places of the app background refresh is implemented with wasLoaded.
Why don't you like this approach?

Reply:
It just doesn’t apply. `wasLoaded` indicates what the name implies: whether data was loaded for the first time or not. This is not enough information to handle the case in which app should behave differently for manual requests and for background automatic requests.

Moreover, if this approach is used in other places in the app, we should redo those places :)

Reply:
Discussed in PM, the issue is specific to this module, in other places `wasLoaded` resets every time data is refetched manually. Will do the same here.